### ISSUE TITLE
`{% attrs %}` do not render attributes starting with a `@`

### ISSUE THREAD

**Issue by jlopinto** [mixxorz/slippers#19]
Hi,

Trying to marry Slippers with AlpineJS I found that attributes starting with `@` do not render.

Component definition:
```html
<button {% attrs type @click %}>{{ children }}</button>
```

Usage:
```twig
{% #button type="button" @click="open = true" %}Open something{% /button %}
```

Render:
```html
<button>Open something</button>
```
It's not even rendering the `type` attribute

**Comment by mixxorz** [comment]
Special character support added in #23.

**Comment by mixxorz** [comment]
I've released version `0.5.0a0` which addresses this issue. Could you please give it a try?

```
pip install slippers==0.5.0a0
```

**Comment by jlopinto** [comment]
I confirm, It now works as expected!
